### ISSUE TITLE
How to provide arguments to `dotnet script exec`?

### ISSUE THREAD

**Issue by mbp** [filipw/dotnet-script#332]
Consider the following script:

```
if (Args.Count > 0)
{
   Console.WriteLine(Args[0]);
}
else
{
   Console.WriteLine("args missing");
}
```

Without publish/exec, I can provide arguments as follows:

```
C:\>dotnet script foo.csx
args missing

C:\>dotnet script foo.csx a
a

C:\>dotnet script foo.csx -- a
a
```

However with publish/exec, I am unable to:

```
C:\>dotnet script publish foo.csx --dll
Published C:\foo.csx to C:\publish\foo.dll

C:\>dotnet script exec publish\foo.dll
args missing

C:\>dotnet script exec publish\foo.dll a
Unrecognized command or argument 'a'

C:\>dotnet script exec publish\foo.dll -- a
args missing
```

**Comment by seesharper** [comment]
@Sharpiro Any thoughts on this?

**Comment by filipw** [comment]
thanks for the report, it was fixed in #333. We will release 0.26.1 soon

**Comment by doggyvufvuf** [comment]
@mbp LETS GO
